rasa_core/policies/fallback.py

```diff
@@ -93,7 +93,10 @@ def predict_action_probabilities(self,
         # if NLU interpreter does not provide confidence score,
         # it is set to 1.0 here in order
         # to not override standard behaviour
-        nlu_confidence = nlu_data["intent"].get("confidence", 1.0)
+        if nlu_data["intent"]:
+            nlu_confidence = nlu_data["intent"].get("confidence", 1.0)
+        else:
+            nlu_confidence = 1.0
```

Comment:
Shouldn't this confidence be `0.0` since there's no recognized intent and we definitely want to be below the NLU threshold so we can NLU fallback? 

Also thinking this should maybe be try/except since it should happen only rarely

Reply:
I think in this case it is okay to predict fallback action with confidence `core_threshold` and not `1.0`. I don't think this changes the behavior here, but if the system learns to respond with a certain action for intent None, the confidence for this intent should be `1.0` and not `0.0`. Does that make sense?